### PR TITLE
[#184726136] Allow setting cromwell version

### PR DESCRIPTION
**Pull request by daniel-thoo**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -85,3 +85,11 @@ variable "allow_deletion" {
 
   description = "Set this to true in order to allow destroying stateful resources (buckets, DBs, etc.)"
 }
+
+variable "cromwell_version" {
+  type     = number
+  nullable = false
+  default = 85
```
Comment (mbarkley) [inline variables.tf:92]: Nitpick: this line is formatted properly.

File: variables.tf [hunk 2]
```diff
@@ -85,3 +85,11 @@ variable "allow_deletion" {
 
   description = "Set this to true in order to allow destroying stateful resources (buckets, DBs, etc.)"
 }
+
+variable "cromwell_version" {
+  type     = number
```
Comment (mbarkley) [inline variables.tf:90]: Cromwell versions are _usually_ numbers, but occasionally they have [hotfix releases](https://github.com/broadinstitute/cromwell/releases/tag/68_hotfix_8e12ab5) that aren't numbers. I'd recommend making this input a string.

**Review by mbarkley — CHANGES_REQUESTED** [review]
@daniel-thoo structurally, this looks good! I have a minor request in-line regarding the version variable type.

**Comment by daniel-thoo** [comment]
@mbarkley Thanks for the quick review! I've addressed your two suggestions. I'll merge these changes.

**Review by mbarkley — APPROVED** [review]

**Pull request merged**